doc/api/http2.md

```diff
@@ -1197,8 +1197,11 @@ added: v8.4.0
     **Default:** `false`
   * `parent` {number} Specifies the numeric identifier of a stream the newly
     created stream is dependent on.
-* `callback` {Function} Callback that is called once the push stream has been
-  initiated.
+* `callback` {Function} Callback that is called once the push stream has been  
+initiated.
```

Comment:
A nit: I wonder if the hard line break is [rendered here](https://github.com/nephross/node/blob/e0eb65c8196bb0126620ff180fbd0d0518466e16/doc/api/http2.md#http2streampushstreamheaders-options-callback) due to missing indentation.